services/worker-manager/src/providers/azure.js

```diff
@@ -387,19 +367,223 @@ class AzureProvider extends Provider {
 
   async scanPrepare() {
     this.seen = {};
+    this.errors = {};
+  }
+
+  /**
+   * Checks the status of ongoing Azure operations
+   * Returns true if the operation is in progress, false otherwise
+   *
+   * op: a URL for tracking the ongoing status of an Azure operation
+   * errors: a list that will have any errors found for that operation appended to it
+   */
+  async handleOperation({op, errors}) {
+    let req, resp;
+    try {
+      // TODO: we could be smarter about getting operation status
```

Comment:
let's get a bug on file for this and add the bugid to this TODO if we're not going to change it before we land this

Reply:
bump ^^ :)

Reply:
I've downgraded this to an `NB` because I'm pretty sure it's inconsequential